### ISSUE TITLE
Unknown keyword's setting definition '[Arguments]'

### ISSUE THREAD

**Issue by nefethael** [nokia/RED#338]
Hi team,

I'm encountering `Unknown keyword's setting definition '[Arguments]'` error for every Keywords having [Arguments] tag. I'm using tab separator format and Red v0.9.0:

Example:

```
| Write Coil
| | [Arguments] | ${reg} | ${value}
| | ${ret}= | Modbus.Write | ${reg} | COIL | ${value}
| | Should Be True | ${ret}
```

Is it a known problem?
Best regards,
Vincent

**Comment by michalanglart** [comment]
We've found out that this issue was reported internally 3 years ago in october 2016 by us however it was down at the botton of backlog since so far no real user complained about this. We'll fix that in next version as it's easy and straightforward.

**Comment by michalanglart** [comment]
Corrected in 0.9.1
